File: Solo Learn/Fácil/Múltiplos/multiples.cpp
#include <iostream>
using namespace std;

int main() {

    int contador = 1;
    int soma = 0;
    int numero;
    
    cin >> numero;
    
    while (contador < numero) // Testando a condição
  {
    if (contador % 3 == 0 || contador % 5 == 0) {
        soma += contador; //atualizando a var soma
    } contador++; //atualizando a var contador
  }   
  
    cout << soma << "\n";
    
    return 0;
}

File: Solo Learn/Fácil/Múltiplos/multiples.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        int numero = scanner.nextInt();

        int contador = 1;
        int soma = 0;

        while (contador < numero) {
            if (contador % 3 == 0 || contador % 5 == 0) {
                soma += contador;
            }
            contador++;
        }

        System.out.println(soma);
    }
}
